test/lib/git-exec.spec.js

```diff
@@ -0,0 +1,9 @@
+//const { expect, assert } = require('chai');
+const git = require('./../../src/lib/git-exec');
+
+describe('git-exec', () => {
+
+  it('execute `git status` do not raise any error', async () => {
+    await git.status();
+  });
+});
```

Comment:
As above, it'd be better to mock out the `exec` call so we can test the `git` unit without actually executing a child process on the machine. This will also mean you can test the `fetch` and `checkUpstream` functions in the `git` module which would otherwise involve a network request.

Reply:
Done.